test/setup/main.tf

```
@@ -18,11 +18,12 @@ module "project" {
   source  = "terraform-google-modules/project-factory/google"
   version = "~> 15.0"
 
-  name              = "ci-genai-knowledge-base"
-  random_project_id = "true"
-  org_id            = var.org_id
-  folder_id         = var.folder_id
-  billing_account   = var.billing_account
+  name                    = "ci-genai-knowledge-base"
+  random_project_id       = "true"
+  org_id                  = var.org_id
+  folder_id               = var.folder_id
+  billing_account         = var.billing_account
+  default_service_account = "deprivilege"
```

The integration test passed with `deprivilege`.  Strange.
Let's try `disable`:
```suggestion
  default_service_account = "disable"
```

---

If this also failed to reproduce the error, try `delete`.